script.sql

```diff
@@ -0,0 +1,162 @@
+
+
+-- tables
+-- Table: app
+CREATE TABLE app (
+    id int NOT NULL AUTO_INCREMENT,
+    name varchar(25) NOT NULL,
+    description text NOT NULL,
+    weight int NOT NULL,
```

Comment:
It's always better to be clear with your naming. `weight` can be measured in kb, mb, gb etc.